spec/stac-geoparquet-spec.md

```diff
@@ -31,11 +31,11 @@ most of the fields should be the same in STAC and in GeoParquet.
 | _property columns_ | _varies_             | -        | Each property should use the relevant Parquet type, and be pulled out of the properties object to be a top-level Parquet field                                                                                                                          |
 
 - Must be valid GeoParquet, with proper metadata. Ideally the geometry types are defined and as narrow as possible.
-- Strongly recommend to only have one GeoParquet per STAC 'Collection'. Not doing this will lead to an expanded GeoParquet schema (the union of all the schemas of the collection) with lots of empty data
+- Recommend to only have one GeoParquet per STAC 'Collection'. Not doing this will lead to an expanded GeoParquet schema (the union of all the schemas of the collection) with lots of empty data
```

Comment:
Maybe rephrase this as strongly recommending that the records be somewhat uniform? That gets to the core if the issue (avoiding a bloated schema, lean on the strengths of parquet), and whether this comes from one or many collections is secondary. So maybe something like

- Strongly recommend that the records be mostly uniform, either because they came from a single STAC collection or multiple STAC collections whose items have similar fields. Apache Parquet is a columnar format, and is most efficient and convenient for users when all the records have the same fields. Not doing this will lead to an expanded GeoParquet schema (the union of all the schemas of the collection) with lots of empty data.

Reply:
And does it make sense to mention the fields extension here?